.storybook/config.js

```diff
@@ -2,6 +2,7 @@ import { configure, addDecorator } from '@storybook/react';
 
 import { backgrounds, StoryWrapper } from './decorators'
 import './styles.scss'
+import '../node_modules/@masterclass/mastersass/src/masterSass.scss'
```

Comment:
This should be here?
You're adding `mastersass` to the development environment but not to `mc-components`.

I think we should import `mastersass` in `src/styles/index.scss` to keep syncronized both libraries.

For example: If I want to install the `mc-components` 0.1.15 version, I just import the scss file from `mc-components` that has the correct `mastersass` version installed.

What do you think?